internal/api/configapi/attestation.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+package configapi
+
+import (
+	"fmt"
+	"net/url"
+	"path"
+	"strings"
+
+	"github.com/edgelesssys/constellation/v2/internal/constants"
+	"github.com/edgelesssys/constellation/v2/internal/variant"
+)
+
+// baseURL is the base URL of the config api.
+var baseURL = constants.CDNRepositoryURL
```

Comment:
Since we dont manipulate this for test anymore, I think the package can use `constants.CDNRepositoryURL` directly